src/garn-bin.js

```diff
@@ -4,9 +4,20 @@ const fs = require('fs');
 const path = require('path');
 const minimist = require('minimist');
 
+const buildsystemPathArgName = 'buildsystem-path';
+if (!process.argv.find(v => v === '--' + buildsystemPathArgName)) {
```

Comment:
```suggestion
if (!process.argv.find(v => v.includes(`--${buildsystemPathArgName}`))) {
```
This would instead handle both `--buildsystem-path="/some/path"` and `--buildsystem-path "/some/path"` being passed.

Reply:
Sure, sounds good to me!

Reply:
Fixed!